Problem accessing private registry

Hi there,

thank you for your great project. I've a problem regarding the access of my private registry. It works if I pass the user and password directly via: `-e REPO_USER=myUser -e REPO_PASS=myPassword`

If I map the configuration via: `-v /root/.docker/config.json:/root/.docker/config.json` there is the following output (exact servername was replaced with server.com):

```
[DEBUG] 2018-11-28 09:52:46 http://localhost:None "GET /v1.35/images/portainer/portainer/json HTTP/1.1" 200 1761
[DEBUG] 2018-11-28 09:52:46 Pulling image: portainer/portainer:latest
[DEBUG] 2018-11-28 09:52:46 Sending supplied auth config
[DEBUG] 2018-11-28 09:52:47 http://localhost:None "POST /v1.35/images/create?tag=latest&fromImage=portainer%2Fportainer HTTP/1.1" 200 None
[DEBUG] 2018-11-28 09:52:47 http://localhost:None "GET /v1.35/images/portainer/portainer:latest/json HTTP/1.1" 200 1761
[DEBUG] 2018-11-28 09:52:47 http://localhost:None "GET /v1.35/images/registry.server.com/server-portfolio:latest/json HTTP/1.1" 200 None
[DEBUG] 2018-11-28 09:52:47 Pulling image: registry.server.com/server-portfolio:latest
[DEBUG] 2018-11-28 09:52:47 Sending supplied auth config
[DEBUG] 2018-11-28 09:52:48 http://localhost:None "POST /v1.35/images/create?tag=latest&fromImage=registry.server.com%2Fserver-portfolio HTTP/1.1" 500 116
[ERROR] 2018-11-28 09:52:48 500 Server Error: Internal Server Error ("Get https://registry.server.com/v2/server-portfolio/manifests/latest: no basic auth credentials")
[INFO] 2018-11-28 09:52:48 0 container(s) updated
```

The `config.json` looks like this:
```
{
        "auths": {
                "registry.codeoftheday.net": {
                        "auth": "dXNlcjpwYXNzd29yZA=="
                }
        },
        "HttpHeaders": {
                "User-Agent": "Docker-Client/18.06.1-ce (linux)"
        }
}
```

Any idea what I could try?

--- a/ouroboros/image.py
+++ b/ouroboros/image.py
@@ -10,7 +10,7 @@ def check_credentials():
     if environ.get('REPO_USER') and environ.get('REPO_PASS'):
         return {'username': environ['REPO_USER'],
                 'password': environ['REPO_PASS']}
-    return {}
+    return None
 
 
 def pull_latest(image, keep_tag, api_client):

--- a/setup.py
+++ b/setup.py
@@ -13,7 +13,7 @@ def read_reqs(requirements):
 
 setup(
     name='ouroboros-cli',
-    version='0.3.1',
+    version='0.3.2',
     description='Automatically update running docker containers',
     long_description=readme(),
     long_description_content_type='text/markdown',

--- a/tests/unit/image_test.py
+++ b/tests/unit/image_test.py
@@ -14,7 +14,7 @@ def test_image_check_credentials_env_var(mocker):
 
 def test_image_check_credentials_false_env_var(mocker):
     mocker.patch.dict('os.environ', {'REPO_USR': 'test_user', 'REPO_PaSS': 'test_pass'})
-    assert image.check_credentials() == {}
+    assert image.check_credentials() == None
 
 
 def test_image_is_up_to_date():

--- a/tests/unit/image_test.py
+++ b/tests/unit/image_test.py
@@ -4,7 +4,7 @@ import docker
 
 
 def test_image_check_credentials_no_env_var():
-    assert image.check_credentials() == {}
+    assert not image.check_credentials()
 
 
 def test_image_check_credentials_env_var(mocker):
@@ -14,7 +14,7 @@ def test_image_check_credentials_env_var(mocker):
 
 def test_image_check_credentials_false_env_var(mocker):
     mocker.patch.dict('os.environ', {'REPO_USR': 'test_user', 'REPO_PaSS': 'test_pass'})
-    assert image.check_credentials() == None
+    assert not image.check_credentials()
 
 
 def test_image_is_up_to_date():

